website/docs/components/link/inline/partials/code/how-to-use.md

```diff
@@ -3,37 +3,47 @@
 The most basic invocation requires some content to be passed as children and either an `@href` or `@route` argument.
 
 ```handlebars
-<Hds::Link::Inline @href="...">Watch tutorial video</Hds::Link::Inline>
+<div class="hds-typography-body-300">
+  Lorem <Hds::Link::Inline @href="...">ipsum dolor</Hds::Link::Inline> sit amet consectetur adipiscing elit.
+</div>
 ```
 
 ### Color
 
-There are two available colors for a Inline Link: `primary` and `secondary`. The default is `primary`. To use a different color, set `@color` to `secondary`.
+There are two available colors for an Inline Link: `primary` and `secondary`. The default is `primary`. To use a different color, set `@color` to `secondary`.
 
 ```handlebars
-<Hds::Link::Inline @color="primary" @href="...">Read tutorial</Hds::Link::Inline>
+<div class="hds-typography-body-300">
+  Lorem <Hds::Link::Inline @color="primary" @href="...">ipsum dolor</Hds::Link::Inline> sit amet consectetur adipiscing elit.
+</div>
 ```
 
 ```handlebars
-<Hds::Link::Inline @color="secondary" @href="...">Read tutorial</Hds::Link::Inline>
+<div class="hds-typography-body-300">
+  Lorem <Hds::Link::Inline @color="secondary" @href="...">ipsum dolor</Hds::Link::Inline> sit amet consectetur adipiscing elit.
+</div>
 ```
 
 ### Icon
 
-To add an icon to your Inline Link, give the `@icon` argument any [icon](/icons/library) name.
+To add an icon to the Inline Link, give the `@icon` argument any [icon](/icons/library) name.
 
 `Hds::Link::Inline` does not have an intrinsic size. Instead, the size of the icon is calculated proportionally (via `em`) in relation to the font-size of the text.
 
 ```handlebars
-<Hds::Link::Inline @href="..." @icon="external-link">Watch tutorial video</Hds::Link::Inline>
+<div class="hds-typography-body-300">
+  Lorem <Hds::Link::Inline @color="primary" @icon="external-link" @href="...">ipsum dolor</Hds::Link::Inline> sit amet consectetur adipiscing elit.
+</div>
 ```
 
 ### Icon position
 
-By default, if you define an icon, it‘s placed in the trailing (right) position. If you would like to position the icon in the leading (left) position, define `@iconPosition`.
+By default, if you define an icon, it‘s placed in the trailing (end) position. If you would like to position the icon in the leading (start) position, define `@iconPosition`.
 
 ```handlebars
-<Hds::Link::Inline @href="..." @icon="film" @iconPosition="leading">Watch tutorial video</Hds::Link::Inline>
+<div class="hds-typography-body-300">
+  Lorem <Hds::Link::Inline @color="primary" @icon="external-link" @iconPosition="leading" @href="...">ipsum dolor</Hds::Link::Inline> sit amet consectetur adipiscing elit.
```

Comment:
What do you think of using a different icon, that works better in a leading position?

Reply:
The `globe` example does seem to be a better solution visually here. I'll update that.